polls/tool/src/bundle/org/sakaiproject/poll/bundle/Messages.properties

```diff
@@ -50,7 +50,9 @@ new_poll_option=Option
 new_poll_submit=Save
 new_poll_saveoption=Save and add options
 new_poll_option_title=Options
-new_poll_option_add=Add option
+new_poll_option_add=Add options one by one
+new_poll_option_add_batch=Add options massively
```

Comment:
"Add options massively" isn't a good translation to English. I'd suggest something like "Add options in bulk" or "Add multiple options". 

"Add options one by one" could be reworded as "Add single option" or leave as "Add option".

Reply:
makes sense to me, thanks